ONSdigital/ras-backstage
# Show correct enrolled respondent for survey for ru

Recreate bug:
1. make up ras-rm-docker-dev repo
2. run acceptance tests from rasrm-acceptance-tests repo
3. Visit a reporting unit page with enrolled user http://localhost:8085/reporting-units/49900000002
4. You will see the enrolled user will be shown for both surveys, but the user only enrolled for Bricks

Test bug fix:

1. Build docker image of this branch docker build . -t sdcplatform/ras-backstage:latest
2. Visit a reporting unit page with enrolled user http://localhost:8085/reporting-units/49900000002
3. You should now only see the respondent in the Bricks section

Acceptance test will break when this is merged in, PR to fix this is here : https://github.com/ONSdigital/rasrm-acceptance-tests/pull/74
